UglyNumber/UglyNumber/UglyNumber.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
These file changes should be ignored by .gitignore
Please update gitignore file.
Please clean up your folder.

Reply:
I'll make a separate pull request cleaning up everything, and then one modifying the gitignore.

Reply:
o.k